static/js/components/dashboard/CourseRow.js

```diff
@@ -45,6 +46,16 @@ export default class CourseRow extends React.Component {
       []
   );
 
+  hasPaidForAnyCourseRun = (course: Course): boolean => (
+    (course.runs.length > 0) ?
+      R.any(R.propEq('has_paid', true), course.runs) :
+      false
```

Comment:
`R.any` is safe to call on `[]`, it will return false in that case (so we don't need the `course.runs.length > 0` check)